pkg/tfvars/tfvars.go

```diff
@@ -8,7 +8,7 @@ import (
 
 type config struct {
 	ClusterID   string `json:"cluster_id,omitempty"`
-	Name        string `json:"cluster_name,omitempty"`
+	ClusterName string `json:"cluster_name,omitempty"`
```

Comment:
I think it would be worthwhile to replace this with `ClusterDomain`. That way the logic for how the cluster domain is constructed is contained in only one place rather than in both go and terraform.

Reply:
done with https://github.com/openshift/installer/pull/1280/commits/3711b75d6bf53e2c7bfd0e15af7d5044b06cebbf